templates/back/signalement/view/information.html.twig

```diff
@@ -571,6 +571,22 @@
                             {{ static_picto_no|raw }}
                         {% endif %}
                     </div>
+                    {% if signalement.isNotOccupant %}

```

Comment:
je n'ai pas l'impression que ce if soit nécessaire, car dans `SignalementBuilder->isDemandeRelogement()`, on met à jour soit en fonction de getLogementSocialDemandeRelogement soit `getInformationsComplementairesSituationOccupantsDemandeRelogement` 
et `isDemandeRelogement` est utilisé pour mettre à jour `isRelogement` (ligne 307)

D'ailleurs, peut-être en profiter pour utiliser aussi `$this->isDemandeRelogement()` ligne 125 au lieu de refaire un `evalBoolean`